src/f32/coresimd/vec3a.rs

```diff
@@ -32,16 +32,16 @@ pub struct Vec3A(pub(crate) f32x4);
 
 impl Vec3A {
     /// All zeroes.
-    pub const ZERO: Self = Self::splat(0.0);
+    pub const ZERO: Self = Self(Simd::from_array([0.0; 4]));
```

Comment:
These should still be able to use `Self::splat` if `Self::splat` is still `const fn` (which `Vec3A::splat` is)

Reply:
I spoke to the maintainers and they had changed splat constness and this was the recommended workaround.